public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -0,0 +1,773 @@
+include ../../../../_includes/_util-fns
+
+:marked
+  There are many conceptual and syntactical differences between Angular&nbsp;1 and Angular&nbsp;2.
+  This chapter provides a quick reference guide to some of the common Angular&nbsp;1
+  syntax and its equivalent in Angular&nbsp;2.
+
+.l-main-section
+:marked
+  ## Template Basics
+  Templates are the user-facing part of an Angular application and are written in HTML.
+  The following are some of the key Angular&nbsp;1 template features with the equivalent
+  template syntax in Angular&nbsp;2.
+
+- var top="vertical-align:top"
+table(width="100%")
+  col(width="50%")
+  col(width="50%") 
+  tr
+    th Angular&nbsp;1
+    th Angular&nbsp;2
+  tr(style=top)
+    td
+      :marked
+        ### Bindings/Interpolation
+      code-example.
+         &lt;h2>{{ vm.title }}&lt;/h2>
+      :marked
+        In Angular&nbsp;1, an expression in curly braces denotes one-way binding.
+        This binds the value of the element to a property in the controller
+        associated with this template
+        
+        When using the `controller as` syntax,
+        the binding is prefixed with the controller alias (vm) because we
+        have to be specific about the source of the binding.
+    td
+      :marked
+        ### Bindings/Interpolation
+      code-example.
+         &lt;h2>{{ title }}&lt;/h2>
```

Comment:
NOTE: The A2 code will be changed to actual code examples once I have confirmation that I'm on the right track with this.